tests/Oryx.RuntimeImage.Tests/NodeImagesTest.cs

```diff
@@ -107,6 +107,38 @@ public void NodeVersionMatchesImageName(string nodeTag, string nodeVersion)
                 result.GetDebugInfo());
         }
 
+        [Theory]
+        [InlineData("4.4")]
+        [InlineData("4.5")]
+        [InlineData("4.8")]
+        [InlineData("6.2")]
+        [InlineData("6.6")]
+        [InlineData("6.9")]
+        [InlineData("6.10")]
+        [InlineData("6.11")]
+        [InlineData("8.0")]
+        [InlineData("8.1")]
+        [InlineData("8.2")]
+        [InlineData("8.8")]
+        [InlineData("8.9")]
+        [InlineData("8.11")]
+        [InlineData("8.12")]
+        [InlineData("9.4")]
+        [InlineData("10.1")]
+        [InlineData("10.10")]
+        [InlineData("10.12")]
+        [InlineData("10.14")]
+        public void NodeImage_Contains_RequiredPrograms(string nodeTag)
+        {
+            // Arrange & Act
+            var result = _dockerCli.Run(
+                $"oryxdevms/node-{nodeTag}:latest",
+                "sh", new[] { "-c", "which tar && which unzip" });
```

Comment:
nit: if you have to split the argument into separate line: we can agree on the following approach(which aspnetcore team follows):
Have all the arguments in the same line, if possible within the length limits
OR
Split each argument into its own separate line

Reply:
I wanted to keep it this way on purpose, so that it would resemble the actual command being run: `sh -c which tar && which unzip`.
WDYT?

Reply:
That is a fair point 👍